includes/Core/Assets/Assets.php

```diff
@@ -373,7 +373,7 @@ private function get_assets() {
 				)
 			),
 			new Script_Data(
-				'googlesitekit-base-data',
+				'googlesitekit-admin-data',
```

Comment:
See above, this should remain `googlesitekit-base-data`.

Reply:
Updated.